### DISCUSSION TITLE
Points, Points, Points  Testnet TEA

### DISCUSSION THREAD

**Discussion by Asunayu25** [teaxyz/white-paper#142] [General]
Points, Points, Points

Testnet is 2 days away! How can you earn points with our Incentivized Testnet? What do the points mean? Why do you want points?

**Comment by dnwandana** [comment]
> Points are what the tea Incentivized Testnet is using to reward participation in the development of the tea Protocol.

src: https://tea.xyz/blog/ultimate-guide-to-points-for-the-tea-itn

> These points have no immediate value but may become redeemable for blockchain tokens in the future.

src: https://tea.xyz/blog/what-to-expect-from-itn

You can read all the blog post at: https://tea.xyz/blog

**Reply by edonor** [reply]
When mainnet

**Comment by Arlo520** [comment]
Points can be redeemed for tea tokens in the future

**Comment by Eriwa00** [comment]
So far , I can say the testnet campaign has been quite smooth and educative being able to learn more about the product. Most of the task are fun and straightforward. 

Lastly when should we be expecting tea protocol to go mainnet.

**Reply by Samped** [reply]
yeah, the testnet have been impressive

**Reply by Eriwa00** [reply]
> yeah, the testnet have been impressive

very true
